frontend_app_html-document/src/container/HtmlDocument.jsx

```diff
@@ -526,7 +533,7 @@ export class HtmlDocument extends React.Component {
       case 400:
         this.setLocalStorageItem('rawContent', backupLocalStorage)
         switch (fetchResultSaveHtmlDoc.body.code) {
-          case 1001:
+          case 2067:
```

Comment:
I think both error codes are possible: 1001 means that the username does not exist, 2067 that the username is not a member of the current space.

Reply:
no, `2067` applies to both.